common/src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/part/ItemBusPartMachine.java

```diff
@@ -149,6 +166,11 @@ public void setWorkingEnabled(boolean workingEnabled) {
     //////////////////////////////////////
     //**********     GUI     ***********//
     //////////////////////////////////////
+
+    public void attachConfigurators(ConfiguratorPanel configuratorPanel) {
+        configuratorPanel.attachConfigurators(new CircuitFancyConfigurator(circuitInventory.storage));
+    }
+
```

Comment:
super.attachConfugurator(configuratorPanel)
if (this.io == IO.IN) {
     configuratorPanel.attachConfigurators(new CircuitFancyConfigurator(circuitInventory.storage));
}